ci-test-connections.yaml

```diff
@@ -28,4 +28,10 @@ connections:
     login: null
     password: null
     port: null
-    schema: null
\ No newline at end of file
+    schema: null
+  - conn_id: sqlite_conn
+    conn_type: sqlite
+    host: ////sqlite.db
```

Comment:
//// - means absolute path. 

So, the final path should be - ////tmp/sqlite.db

I was trying with ////sqlite.db, since that's the only dir I knew exist, seems like CI user don't have permission to create a file there.